server/pulp/server/util.py

```diff
@@ -2,16 +2,42 @@
 This module contains utility code to be used by pulp.server.
 """
 from contextlib import contextmanager
+from gettext import gettext as _
+import hashlib
 import logging
 import os
 from shutil import copy, Error
 
-from gettext import gettext as _
+from pulp.common import error_codes
+from pulp.server.exceptions import PulpExecutionException, PulpCodedException
```

Comment:
Also consider switching the order of `PulpExecutionException` and `PulpCodedException` so they are alphabetical. These are such small things it's even silly to write it.